library/core/src/primitive_docs.rs

```diff
@@ -1628,8 +1628,8 @@ mod prim_ref {}
 /// let ptr: fn(usize) -> usize = add_one;
 /// assert_eq!(ptr(5), 6);
 ///
-/// let clos: fn(usize) -> usize = |x| x + 5;
-/// assert_eq!(clos(5), 10);
```

Comment:
I don't love having to change this, abbreviations get used for identifiers a lot.

Reply:
One small public docstring with example.

Reply:
Not checking four letter words would avoid needing this change.